### ISSUE TITLE
Build portable-atomic failed on local ubuntu21 pc 

### ISSUE THREAD

**Issue by ygf11** [taiki-e/portable-atomic#19]
## os ##
Linux ubuntu21 5.11.0-49-generic #55-Ubuntu SMP Wed Jan 12 17:36:34 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

## build command ##
sh tools/build.sh

## error log ##
```
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/tmp/rustcLBmdNm/symbols.o" "/home/work/others/taiki-e/portable-
atomic/target/assume-single-core/debug/build/portable-atomic-b115f9d0a2ef6f1e/build_script_build-
...
"/root/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-
gnu/lib/liblibc-9dba3f083c9eb142.rlib" "/root/.rustup/toolchains/nightly-x86_64-unknown-linux-
gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-dfc71beee59013ce.rlib" 
"/root/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-
gnu/lib/librustc_std_workspace_core-522518611024dce5.rlib" "/root/.rustup/toolchains/nightly-x86_64-
unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-05898138a596088a.rlib" "-Wl,--
end-group" "/root/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-
linux-gnu/lib/libcompiler_builtins-6f7f4877ee9c613e.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt"
 "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" 
"/root/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib"
 "-o" "/home/work/others/taiki-e/portable-atomic/target/assume-single-core/debug/build/portable-
atomic-b115f9d0a2ef6f1e/build_script_build-b115f9d0a2ef6f1e" "-Wl,--gc-sections" "-pie" "-Wl,-
zrelro,-znow" "-nodefaultlibs"
  = note: cc: fatal error: cannot read spec file './specs': Is a directory
          compilation terminated.
          

error: could not compile `portable-atomic` due to previous error
warning: build failed, waiting for other jobs to finish...
error: process didn't exit successfully: `/root/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo clippy -Z check-cfg=names,values,output,features --target avr-unknown-gnu-atmega328 -Z build-std=core,alloc --target-dir target/assume-single-core --manifest-path Cargo.toml --no-default-features` (exit status: 101)
```

## other info ##
1. Build command will success when I reset `portable-atomic` to `release-0.3.4`(91a3e4ec73e738511e7f29e8c1d432ab70e8ede8).

2. It will build failed with the same error, when I build [databend](https://github.com/datafuselabs/databend) and [opendal](https://github.com/datafuselabs/opendal) which also use `portable-atomic`.

Relative: datafuselabs/databend#6823

**Comment by taiki-e** [comment]
Thanks for the report! The existence of a directory called `specs` seems to be causing the problem. I renamed it in the latest main (https://github.com/taiki-e/portable-atomic/commit/0e473062ee90d67f5a1ad8eb6184be6675b1e909), could you check if it fixed the problem?

**Comment by ygf11** [comment]
> could you check if it fixed the problem?

Thanks for quick fix, I can build it now :D.

But other apps which use `portable-atomic` still can not build, maybe we need publish it.

**Comment by taiki-e** [comment]
Thanks for confirming! I've published the fix in 0.3.6.
